tests/foreman/api/test_host.py

```diff
@@ -1349,3 +1351,145 @@ def test_positive_verify_files_with_pxegrub2_uefi_secureboot(self):
 
         @caselevel: System
         """
+
+
+class HostInterfaceTestCase(APITestCase):
+    """Tests for Host Interfaces"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create a host to reuse in tests"""
+        super(HostInterfaceTestCase, cls).setUpClass()
+        cls.host = entities.Host().create()
+
+    @tier1
+    def test_positive_create_with_name(self):
+        """Create an interface with different names and minimal input
+        parameters
+
+        @id: a45ee576-bec6-47a6-a018-a00e555eb2ad
+
+        @Assert: An interface is created with expected name
+        """
+        for name in valid_interfaces_list():
+            with self.subTest(name):
+                interface = entities.Interface(
+                    host=self.host,
+                    name=name,
+                ).create()
+                self.assertEqual(interface.name, name)
+
+    @tier1
+    def test_negative_create_with_name(self):
+        """Attempt to create an interface with different invalid entries as
+        names (empty name, >255 chars, unsupported string types)
+
+        @id: 6fae26d8-8f62-41ba-a1cc-0185137ef70f
+
+        @Assert: An interface is not created
+        """
+        for name in invalid_interfaces_list():
+            with self.subTest(name):
+                with self.assertRaises(HTTPError):
+                    entities.Interface(
```

Comment:
1. There's no strict rule for that (or at least i'm not aware of any), it's just personal preference - i prefer to place each key-value pair from the new line when there is more than 1 pair. Imo it adds readability a lot, although it may be not that noticeably for just 2 pairs. But it's just personal preference, if you don't like it i'm totally ok to change it.